### ISSUE TITLE
Add Polaroid Cameras

### ISSUE THREAD

**Issue by StrawberryMoses** [unitystation/unitystation#2317]
# Description

Add polaroid cameras. These can be used to take static images of a scene on the station. After taking a picture, a photo is printed out as an item.

Notes and considerations:

- The flash can be turned on or off. If the flash is on, the picture is set to have a higher brightness if the lights are off.
- Alt click changes it's focus, determining how big of an area the camera takes. This could also be a right click function.
- Cameras can use film or holodisks, but for a start film is fine. Every picture taken reduces the film's count by 1. Examining the camera shows how much film is left.
- There is a cooldown while the picture is being developed while no more photos can be taken.
- The picture includes a description of all mobs in it.
- When printed, photos can be customized with a name, a description, and a caption.
- Photos have a flag set if the station blueprints are in it, which allows traitors to greentext if that's their objective.

https://github.com/tgstation/tgstation/blob/aa8e09ed021228c1e77bffeb28d7f033ab53c702/code/modules/photography/camera/camera.dm

## Category
TODO

**Comment by fomalsd** [comment]
how do you suggest actually taking photos? simply rendering into a texture on serverside and sending to observing clients in binary form?

**Comment by StrawberryMoses** [comment]
Yeah, I think just making an actual image like that might be easiest. There's also a photo album object that saves pictures through different rounds, so it might be worth planning it like that so they can be kept even with multiple versions of the game
